apps/tsilhqotin-dpb-expo/App.js

```diff
@@ -0,0 +1,23 @@
+import { StatusBar } from 'expo-status-bar'
+import { StyleSheet, Text, View } from 'react-native'
+import { WebView } from 'react-native-webview'
+
+export default function App() {
+    return (
+        <WebView
+            source={{ uri: 'http://testing.tsilhqotinlanguage.ca' }}
```

Comment:
config?

Reply:
I agree- but don't worry about the mobile app for now. Focus on using a config for the web project.